jquery.flowchart.js

```diff
@@ -210,16 +210,17 @@ $(function() {
             if (!multipleLinksOnOutput || !multipleLinksOnInput) {
                 for (var linkId2 in this.data.links) {
                     var currentLink = this.data.links[linkId2];
+                    var currentOperator = this.data.operators[currentLink.fromOperator];
```

Comment:
Suggestion: replace the name of the variable to fromOperator, and add "toOperator = this.data.operators[currentLink.toOperator];", because of my remark below...